analysis_scripts/qc_scripts/pipeline_qc.md

```diff
@@ -67,7 +67,9 @@ Now lets do a quick plot of article specific stats. So far in the pipeline, we o
 
 #### plotting quote stats
 
-Between pipeline step 3 and 4 we are predicting the genders of speakers using genderize.io. So we expect almost exactly the same number of quotes and length of quotes. We also expect that the number of UNKNOWN gendered speakers typically decrease, and the number of MALE/FEMALE speakers may increase. This is not a completely 1:1 measurement. Pipeline level 3 only identifies the number of male/female/unknown named entities, there is no quote attribution checked at this stage. Quote attribution is in step 4. <img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-1.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-2.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-3.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-4.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-5.png" width="50%" />
+Between pipeline step 3 and 4 we are predicting the genders of speakers using genderize.io. So we expect almost exactly the same number of quotes and length of quotes. We also expect that the number of UNKNOWN gendered speakers typically decrease, and the number of MALE/FEMALE speakers may increase. This is not a completely 1:1 measurement. Pipeline level 3 only identifies the number of male/female/unknown named entities, there is no gender based quote attribution checked at this stage. Quote attribution is in step 4.
```

Comment:
If between steps 3 and 4 you are predicting the genders of speakers, why would you expect `almost exactly`` the same number of quotes? Why not exactly? Are quotes for which speaker's gender cannot be predicted discarded?

Reply:
Yes, its a mis-type. You expect the _exact_ same number of quotes and _almost_ exact same length of quotes, since some string cleaning is done in step 4 (unicode fixes + whitespace removal). Language is fixed here: 3fec287